instrument/Facilities.xml

```diff
@@ -684,10 +684,6 @@
     <technique>SANS</technique>
   </instrument>
 
-  <instrument name="D11B">
```

Comment:
Can I just double check that this is intentional that these instruments are removed?

Reply:
Yes, that was the point. The IDFs are there of course, but they shouldn't be listed in facilities as separate from D11 and D22.

Reply:
Ah yes I see now. Thanks for clarifying.